composer.json

```diff
@@ -28,7 +28,7 @@
 		"libphutil": "1.2"
 	},
 	"require-dev": {
-		"behat/behat": "~3.0",
+		"behat/behat": "v3.1.0rc1",
```

Comment:
That is a bit harsh.
Wouldn't `~3.1` make composer install this 3.1.0 RC given https://github.com/Behat/Behat/blob/master/composer.json#L50 ? That is also not perfect as this would basically install master version, so I am not sure what is a better way.
So I am actually not opposing here.